example/plugins/c-api/cache_scan/cache_scan.cc

```diff
@@ -33,10 +33,12 @@
 #include "ts/ts.h"
 #include "ts/experimental.h"
 #include "tscore/ink_defs.h"
+#include "tscpp/api/Cleanup.h"
 
 #define PLUGIN_NAME "cache_scan"
 
-static const TSDbgCtl *const dbg_ctl = TSDbgCtlCreate(PLUGIN_NAME);
+static atscppapi::TSDbgCtlUniqPtr dbg_ctl_guard{TSDbgCtlCreate(PLUGIN_NAME)};
+static const TSDbgCtl *const dbg_ctl{dbg_ctl_guard.get()};
```

Comment:
namespace?

Reply:
mission creep

Reply:
How so? Are you going to fix it in a future PR?

Reply:
How much of what is currently not ideal about ATS do I personally need to commit to fixing so I can get the PR approved?  What's next, are my relatives going to be kidnapped also?

Reply:
Do you mean, if a file currently only uses `static` for file scope, I should nonetheless use an anonymous namespace for new DbgCtl file scope objects.  I'll do that if @SolidWallOfCode concurs.  Hard to anticipate what color bikeshed everyone wants beforehand.

Reply:
I only mentioned it because you have done so in other PRs.